mkdocs.yaml

```diff
@@ -80,3 +80,6 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
   - toc:
       permalink: "#"
+
```

Comment:
[style]
- No unnecessary new line.
- Sort alphabetically.

```suggestion
```

Reply:
FYI: Maybe adding sync settings like this is useful.
https://github.com/autowarefoundation/open-ad-kit-docs/blob/0c1c70625fb89d144a5e2d5df52d0cdccd6e18c4/.github/sync-files.yaml#L37-L41
https://github.com/autowarefoundation/autoware-documentation/blob/fa986159d4fd83d81f232f5ec1e657e5685f495f/mkdocs.yaml#L67-L69

Reply:
I added sync in https://github.com/autowarefoundation/autoware.universe/pull/3620, and https://github.com/autowarefoundation/autoware.universe/pull/3619 is generated.

Would my these changes be as you intended?

Reply:
Yes, exactly. Thank you!

Reply:
Thanks for your comments!